backend/src/server/routes/v1/dashboard-router.ts

```diff
@@ -17,6 +17,20 @@ import { AuthMode } from "@app/services/auth/auth-type";
 import { SecretsOrderBy } from "@app/services/secret/secret-types";
 import { PostHogEventTypes } from "@app/services/telemetry/telemetry-types";
 
+// handle querystring boolean values
+const booleanSchema = z
+  .union([z.boolean(), z.string().trim()])
+  .transform((value) => {
+    if (typeof value === "string") {
+      // ie if not empty, 0 or false, return true
+      return !!value && Number(value) !== 0 && value.toLowerCase() !== "false";
```

Comment:
This should be an OR gate right?

Also `!!value` => Boolean(value)

Reply:
Will swap to `Boolean(value)` but why would should this be an OR gate? 

it's essentially saying return `true` only if not "" and "0" and "false",  which all of these should evaluate true for it to be `true`?